NOTICE

```diff
@@ -1,120 +1,574 @@
-  Jetty :: Server Core under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Servlet Handling under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Webapp Application Support under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Websocket :: API under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Websocket :: Client under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Websocket :: Common under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Websocket :: Server under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: Websocket :: Servlet Interface under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty :: XML utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty Server under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  Jetty Utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
-  JSch under BSD
-  Kryo Shaded under 3-Clause BSD License
-  leveldbjni-all under The BSD 3-Clause License
-  Metrics Core under Apache License 2.0
-  Metrics Core Library under Apache License 2.0
-  MinLog under New BSD License
-  Netty/All-in-One under Apache License, Version 2.0
-  Objenesis under Apache 2
-  org.jetbrains.kotlin:kotlin-stdlib under The Apache License, Version 2.0
-  org.jetbrains.kotlin:kotlin-stdlib-common under The Apache License, Version 2.0
-  org.jetbrains.kotlin:kotlin-stdlib-jdk7 under The Apache License, Version 2.0
-  org.jetbrains.kotlin:kotlin-stdlib-jdk8 under The Apache License, Version 2.0
-  Protocol Buffer Java API under New BSD license
-  RocksDB JNI under Apache License 2.0 or GNU General Public License, version 2
-  servlet-api under CDDL
-  SLF4J API Module under MIT License
-  SLF4J LOG4J-12 Binding under MIT License
-  Spark Integration for Kafka 0.8 under Apache 2.0 License
-  temp under 3-Clause BSD License
-  The Netty Project under Apache License, Version 2.0
-  Xerces2 Java Parser under The Apache Software License, Version 2.0
-  XML Commons External Components XML APIs under The Apache Software License, Version 2.0
-  Xml Compatibility extensions for Jackson under The Apache Software License, Version 2.0 or GNU Lesser General Public License (LGPL), Version 2.1
-  xmlenc Library under The BSD License
-  XZ for Java under Public Domain
-  ZkClient under The Apache Software License, Version 2.0
-  zookeeper under Apache License, Version 2.0
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
You are voting a source release based on what is available here
https://dist.apache.org/repos/dist/dev/incubator/hudi/hudi-0.5.0-incubating-rc2/

All the below is not necessary...
See an example of Spark: https://github.com/apache/spark/blob/master/NOTICE

Reply:
@lresende : My understanding was that the bundled jars is also a distribution and any bits that are included in a distribution have bearing on the content of NOTICE.  Isn't it not the case ? Do we need separate NOTICE-binary file for jars. Please clarify

Reply:
@lresende : Looked at other incubating projects and they all have top-level NOTICE setup similar to the way you mentioned i.e only those that are part of source release. I have fixed them now. Please take a look